packages/reduce/module.d.ts

```diff
@@ -1,2 +1,12 @@
-export type Reducer<T> = (previous: T, current: T, index: number, arr: T[]) => T;
-export default function<T>(arr: T[], callback: Reducer<T>, initialValue?: T): T;
+export type Reducer<T, R> = (
+	previous: R,
+	item: T,
```

Comment:
I'd prefer this to be `current` if that's ok
```suggestion
	current: T,
```